lib/proxy/oauth/signatures/oauth1a.js

```diff
@@ -40,9 +40,9 @@ function normaliseRequestParams(argument_pairs) {
       continue;
     }
     **/
-    args += sprintf("%s=%s", argument_pairs[i][0], argument_pairs[i][1]);
+    args += sprintf("%s%%3D%s", argument_pairs[i][0], argument_pairs[i][1]);
```

Comment:
The more I look at it, I think this line is wrong (both before and after edit).  A query param with no value and no equals is common and (I think) legal, but this code assumes there's always an equals.  This needs more testing and potentially a fix.